pkg/helper/pipeline/utils.go

```diff
@@ -0,0 +1,50 @@
+// Copyright © 2019 The Tekton Authors.
```

Comment:
Rename file according to functionality or something instead of generic name? Like last_pipelinerun.go .

Reply:
Changed to pipelinelastrun.go and pipelinelastrun_test.go